src/ui/public/doc_table/components/table_row/cell.html

```diff
@@ -1,7 +1,7 @@
 <%
 var attributes = '';
 if (timefield) {
-  attributes='class="discover-table-timefield" width="1%"';
+  attributes='class="discover-table-timefield"';
```

Comment:
I wish we could get rid of the special treatment here as well. This class seems to just set the `white-space: nowrap` style - which would make sense on other date columns as well. Unfortunately, a simple solution does not come to mind right now... :thinking: it bears some resemblance with https://github.com/kobelb/kibana/blob/038f6cae28687cfdbb2aa215cff77b0174e3f9c5/src/ui/public/doc_table/components/table_row.js#L180, even though that is concerned with the height.